### PR TITLE
Fix: reactstrap tooltips causing browser tab to freeze

### LINKED ISSUE #4853: Search results page causes Sourcegraph tab to freeze in Firefox
- **Sourcegraph version:** k8s.sgdev.org
- **Platform information:** macOS, **Firefox 68.0 (64-bit)**

#### Steps to reproduce:

1. Open https://k8s.sgdev.org/search?q=route533+index:only
2. See no results
3. Wait for 5+ seconds

#### Expected behavior:

I can continue using the site, by clicking links in the UI for example

#### Actual behavior:

Site freezes, can't click on links anymore, Firefox shows a warning "this web page appears to be slow"

#### Debugging info

This appears to only happen on the results page and it seems like only when it's empty

I tried to capture a performance profile in Firefox but the browser froze when trying to save it. What I _could_ see before saving was a lot of calls to `setInterval`.

Since the status messages for the global cloning indicator are polled, I disabled that feature to see if it's the reason for the freeze. But even with that disabled and no requests happening in the background, the site freezes.

(I don't know who to assign this too, feel free to reassign @nicksnyder @christinaforney)

It also seems to be related to https://github.com/sourcegraph/sourcegraph/issues/4755

### PR DESCRIPTION
**Pull request by attfarhan**
Fixes #4853. 

The commit https://github.com/sourcegraph/sourcegraph/commit/d9e6207cb7790db2a5fa13e90af4423c5cb47299 resulted in a bug that would cause the entire page to freeze if the tooltip popovers that appear when hovering on some buttons had too much content, which is a [bug in the upgraded version of reactstrap](https://github.com/reactstrap/reactstrap/issues/1482). This PR implements a [workaround mentioned in the issue, which disables the `flip` modifier of the tooltip](https://github.com/reactstrap/reactstrap/issues/1482#issuecomment-499729774).

### CODE REVIEW

**Comment by attfarhan** [comment]
The issue is filed in reactstrap here: https://github.com/reactstrap/reactstrap/issues/1482

**Comment by attfarhan** [comment]
Alternatively, we can use a [workaround that was mentioned in the issue](https://github.com/reactstrap/reactstrap/issues/1482#issuecomment-498747688). Pushed that commit up here: https://github.com/sourcegraph/sourcegraph/commit/b65c7de992611adc609c38e40625abffbab551ea

**Comment by attfarhan** [comment]
@felixfbecker there's quite a number of places we use tooltips that could potentially be affected: https://sourcegraph.com/search?q=r:%5Egithub.com/sourcegraph/sourcegraph%24+data-tooltip
Didn't go through all of them. 

There's actually a slightly better workaround in that issue. I think that's the workaround we should implement, actually. https://github.com/sourcegraph/sourcegraph/commit/7f2671eec79a89ab88fa84101c73c938cc9cd0bf

**Comment by attfarhan** [comment]
@felixfbecker the flip modifier tries to flip the direction of the popover if it's too close to the edge of the container. It's on by default, and seems to be what causes the tooltip to freak out as it tries to flip in an infinite loop.

File: web/src/components/tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -67,6 +67,11 @@ export class Tooltip extends React.PureComponent<Props, State> {
                 isOpen={true}
                 target={this.state.subject}
                 placement="auto"
+                modifiers={{
+                    flip: {
+                        enabled: false,
+                    },
+                }}
```
Comment (felixfbecker) [inline web/src/components/tooltip/Tooltip.tsx]: When adding a workaround like this, add a comment linking to the issue so we know to remove it when the issue is fixed

**Pull request merged**